package.json

```diff
@@ -33,7 +33,7 @@
 	"scripts": {
 		"version": "yarn run bootstrap && yarn build",
 		"bootstrap": "lerna bootstrap",
-		"prebuild": "yarn clean",
+		"prebuild": "yarn clean && ts-node scripts/init.ts",
```

Comment:
This will create a `.secrets.json` file from the template if it doesn't already exist when installing the project - needed so the import in `packages/web3/test/e2e/e2e_utils.ts` doesn't cause a `file doesn't exist` error